UI/CUIWnd.cpp

```diff
@@ -633,32 +634,32 @@ void CUIWnd::SetDefaultedOptions() {
 
 void CUIWnd::SaveDefaultedOptions() {
     OptionsDB& db = GetOptionsDB();
-    std::string config_prefix = "UI.windows." + m_config_name;
-    std::string windowed = "";
-    if (!db.Get<bool>("fullscreen"))
-        windowed = "-windowed";
+    std::string config_prefix = "ui." + m_config_name;
+    std::string windowed = ".fullscreen";
```

Comment:
?

Reply:
I don't understand the question ;)
Would the following address the concern: `std::string window_mode = db.Get<bool>("video.fullscreen.enabled") ? ".fullscreen" : ".windowed";`, or is the code unclear and need commenting?

Reply:
It seems strange to have a string variable named `windowed` contain the text `fullscreen`. The suggested modification is better.